### ISSUE TITLE
Error with themeConfiguration Params

### ISSUE THREAD

**Issue by jonyr** [datatrans/react-datatrans-light-box#8]
Hi guys! Me again.
I found a new issue when passing themeConfiguration param as Lightbox props.

```
const settings = {
            useAlias: true,
            < ... other params ...>
            themeConfiguration: { brandColor: '#444444', logoType: 'none', brandTitle: 'none' },
        };

<Ligthbox ...settings />
```

And the requested url to raise the lightbox modal is

`https://pilot.datatrans.biz/upp/jsp/upStart.jsp?useAlias=true&<...other params...>&themeConfiguration=%5Bobject%20Object%5D`

I think the problem is again in src/utils.js line 18 => export const **toUrlParams**. The function doesn't know how to handle the themeConfiguration object.

Could someone help me with this?
Thanks in advance guys.

You really did a great work.

**Comment by jonyr** [comment]
Hi Guys! I could handle the issue using JSON.stringify. So, discard the open issue.

**Comment by pstadler** [comment]
Thanks for your report. This has been fixed with [1.0.9](https://github.com/datatrans/react-datatrans-light-box/releases/tag/1.0.9). 

Please note that this shouldn't break your existing workaround, but it's no longer necessary to call `JSON.stringify` yourself.
